docs/concierge-bundles.md

```diff
@@ -23,6 +22,3 @@ Provides an implementation of the `StartLevel` service.
 
 Allows any JAXP compliant XML Parser to register itself as an OSGi parser service.
 
-### org.eclipse.concierge.service.eventadmin
-
-Provides a very lightweight implementation of EventAdmin (just 3 clasees).
```

Comment:
I think the docs you used have been outdated docs from Website. We should use the version from the old "docs" folder